packages/cmrjs/tests/test-index.js

```diff
@@ -115,3 +115,30 @@ test('get CMR metadata, fail', async (t) => {
 
   stub.restore();
 });
+
+test('searchConcept handles paging correctly.', async (t) => {
+  const stub = sinon.stub(got, 'get');
+  const headers = { 'cmr-hits': '6' };
+  const body1 = '{"feed":{"updated":"sometime","id":"someurl","title":"fake Cmr Results","entry":[{"cmrEntry1":"data"}, {"cmrEntry2":"data2"}]}}';
+  const body2 = '{"feed":{"updated":"anothertime","id":"another url","title":"more Results","entry":[{"cmrEntry3":"data3"}, {"cmrEntry4":"data4"}]}}';
+  const body3 = '{"feed":{"updated":"more time","id":"yet another","title":"morer Results","entry":[{"cmrEntry5":"data5"}, {"cmrEntry6":"data6"}]}}';
+
+  stub.onCall(0).returns({ body: body1, headers: headers });
+  stub.onCall(1).returns({ body: body2, headers: headers });
+  stub.onCall(2).returns({ body: body3, headers: headers });
+
+  const expected = [
+    { cmrEntry1: 'data' },
+    { cmrEntry2: 'data2' },
+    { cmrEntry3: 'data3' },
+    { cmrEntry4: 'data4' },
+    { cmrEntry5: 'data5' },
+    { cmrEntry6: 'data6' }
+  ];
+
+  const results = await searchConcept('collections', {});
+
+  t.deepEqual(expected, results);
```

Comment:
This test is assuming that the results are guaranteed to be returned in a certain order, which I don't think is the case.  The order could change, and still be a perfectly correct response, but the test would then fail.